docs/snippets/react/button-story-component-args-primary.js.mdx

```diff
@@ -6,7 +6,6 @@ import React from 'react';
 import Button from './Button';
```

Comment:
```suggestion
import { Button } from './Button';
```

Reply:
This also applies to the .ts version of this file, but no changes were detected to tag. I'll leave that to you 😄